docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -164,3 +164,257 @@ The test results indicate that the minimum amount of memory for running {Product
 ** Installing the cluster monitoring tools requires an additional 5 GB of memory.
 ====
 
+== Red Hat Single Sign-On
+{ProductShortName} creates a basic
+https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6[Red
+Hat Single Sign-On] instance, using the Keycloak CR of Red Hat Single
+Sign-On Operator. Keycloak is a single sign-on solution for web apps and
+RESTful web services.
+
+The {ProductShortName} Keycloak realm is fully configured by the Hub.
+
+{ProductShortName} does not enable RHSSO external access by default. However, if you
```

Comment:
What does "enable external access for RHSSO" mean?

Reply:
@rromannissen It means that we can request the rhsso operator to actually create a route and expose SSO for the customer to configure via web console , by default is off.

Reply:
@fbladilo thanks for the clarification. i will get this added.

Reply:
@CarmiWisemon - how about we try something like:

{ProductShortName} does not enable RHSSO external access by default. The RHSSO operator can create a route and expose single sign-on for you to configure using the web console.